### DISCUSSION TITLE
关于静止帧的问题

### DISCUSSION THREAD

**Discussion by dugufly** [Tencent/libpag#1783] [Q&A] [answered]
如果在AE中，某个图片图层在时间维度一直没有变化， 引擎里有一个静止帧的概念会避免重复刷新吧？ 如果对这个图层用纹理替换， 纹理的内容持续有变化，这个静止帧会不会影响纹理的刷新？

**Comment by kevingpqi123** [comment, marked as answer]
这属于内部缓存实现相关，如果渲染的内容相对比上一帧没有发生变化，会使用上一次的缓存的内容，逐帧用纹理替换意味着渲染内容发生了变化，内部也会检测到这种变化，如果调用了 flush 方法，是会重新绘制的
